winter/django.py

```diff
@@ -109,6 +110,15 @@ def _rewrite_uritemplate_with_regexps(winter_url_path: str, methods: List[Contro
         if len(types) > 1:
             raise Exception(f'Different methods are bound to the same path variable, but have different types annotated: {types}')
         type_, = types
-        regexp = r'\d+' if issubclass(type_, int) else r'\w+'
+        regexp = _get_regexp(type_)
         url_path = url_path.replace(f'{{{variable_name}}}', f'(?P<{variable_name}>{regexp})')
     return url_path
+
+
+def _get_regexp(type_) -> str:
+    if issubclass(type_, int):
```

Comment:
It is better to use registering types here
for example for date
for datetime
for slug
for something like that like in django 
https://docs.djangoproject.com/en/2.1/topics/http/urls/
see register_converter